ledger/sandbox/src/test/lib/scala/com/digitalasset/platform/sandbox/LedgerResource.scala

```diff
@@ -62,6 +62,7 @@ object LedgerResource {
               ledgerId,
               timeProvider,
               Nil,
+              2,
```

Comment:
```suggestion
              128,
```

Reply:
`LedgerResource` is currently only used in a single test. I'd rather not restart the whole CI shenanigans but get master back to green.

Reply:
Ok, I will change it in a follow up PR then.